sql/updates/latest-dev.sql

```diff
@@ -47,3 +47,70 @@ LANGUAGE C VOLATILE;
 DROP PROCEDURE IF EXISTS _timescaledb_functions.policy_compression_execute(job_id INTEGER, htid INTEGER, lag ANYELEMENT, maxchunks INTEGER, verbose_log BOOLEAN, recompress_enabled  BOOLEAN, use_creation_time BOOLEAN);
 
 DROP PROCEDURE IF EXISTS _timescaledb_functions.policy_compression(job_id INTEGER, config JSONB);
+
+CREATE PROCEDURE @extschema@.convert_to_columnstore(
+    chunk REGCLASS,
+    if_not_columnstore BOOLEAN = true,
+    recompress BOOLEAN = false,
```

Comment:
```suggestion
    rebuild BOOLEAN = false,
```
WDYT?

Reply:
That is a good suggestion, it makes it clear that it is an expensive operation and is still terse. Since this is not an option that the user normally uses, she would have to check the manual anyway.

Reply:
Yes, we should not have `recompress` as the name of this parameter since it is unclear what it means. Using `rebuild` is better but still it is not clear what it means (e.g., apply new settings).

I would also consider removing it altogether since the same thing can be achieved with a decompress+compress, which might actually be better since that can be two separate transactions and thus more control over locks. We can always add back later.